practice/pyramid/site.css

```diff
@@ -0,0 +1,22 @@
+.row {
+  box-sizing: border-box;
+  height: 100px;
+  padding: 20px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.inner {
+  box-sizing: border-box;
+  height: 100px;
+  padding: 10px;
+  flex: 1;
+}
+
+.long {
+  flex: 4;
+}
+
+.inner.threecolumns {
```

Comment:
These three boxes are not laid out like the example. There needs to be no space on the outsides.